src/Config/CacheFileLocator.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Config;
+
+use Symfony\Component\Config\FileLocatorInterface;
+
+/**
+ * Searches the cache paths for files.
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+class CacheFileLocator implements FileLocatorInterface
```

Comment:
This class name is wrong. The class does not know about cache, it simply knows about paths. It's simply a `FileLocator`.